lib/oli_web/controllers/page_delivery_controller.ex

```diff
@@ -342,6 +343,8 @@ defmodule OliWeb.PageDeliveryController do
          _,
          _
        ) do
+    session_context = SessionContext.init(conn)
```

Comment:
Actually I decided to use `session_context` because this function also receives a `%PageContext{}` under a variable named `context`, and I thought it would be cleaner to differentiate them.

Reply:
Ooooh missed that `context` var sorry. Let's pretend i didn't say anything 😃